compiler/luci/pass/src/ResolveCustomOpBatchMatMulPass.cpp

```diff
@@ -40,10 +42,17 @@ bool resolve_custom_op(luci::CircleCustom *cop)
     batch_matmul->adj_x(map["adj_x"].AsBool());
     batch_matmul->adj_y(map["adj_y"].AsBool());
 
-    replace(cop).with(batch_matmul);
-    return true;
+    for (auto s : loco::succs(cop))
+    {
+      if (auto cop_out = dynamic_cast<luci::CircleCustomOut *>(s))
+      {
+        replace(cop_out).with(batch_matmul);
+        changed = true;
+      }
+    }
   }
-  return false;
+
+  return changed;
```

Comment:
```suggestion
  return true;
```

Reply:
@mhs4670go , `return true` seems to make infinite run. maybe set `changed = true` inside `if` block?